politeiawww/api/v1/v1.go

```diff
@@ -694,7 +696,8 @@ type NewProposalReply struct {
 // ProposalsDetails is used to retrieve a proposal.
 // XXX clarify URL vs Direct
 type ProposalsDetails struct {
-	Token string `json:"token"`
+	Token   string `json:"token"`
```

Comment:
Struct fields in `v1.go` should be commented for documentation purposes.  There are a few structs that don't have their fields commented, but most of them do, so lets make sure new code is commented properly.

The `ProposalDetails` comment also needs to be updated to include the fact that `Version` is a optional query param and that the latest version of the proposal will be returned when no `Version` param is included.